src/logic/gameReducer.js

```diff
@@ -130,8 +130,23 @@ function dragStart({
     return currentGameState;
   }
 
-  // Find the top left of the group in client coordinates, to get pointerOffset.
-  if (pointerOffset === undefined) {
+  let pointerOffset;
+  if (previousDragState && previousDragState.pieceIDs.length == 1) {
+    // Use previous pointerOffset, adjusted for the different group of pieces we have now.
+    const previousPiece = currentGameState.pieces.filter(
+      (piece) => piece.id == previousDragState.pieceIDs[0]
+    )[0];
+    const extraSquaresLeft = previousPiece.boardLeft - groupBoardLeft;
+    const extraSquaresTop = previousPiece.boardTop - groupBoardTop;
+    const boardRect = document.getElementById("board").getBoundingClientRect();
+    const squareWidth = (boardRect.width - 1) / currentGameState.gridSize;
+    const squareHeight = (boardRect.height - 1) / currentGameState.gridSize;
+    pointerOffset = {
+      x: previousDragState.pointerOffset.x + squareWidth * extraSquaresLeft,
+      y: previousDragState.pointerOffset.y + squareHeight * extraSquaresTop,
+    };
```

Comment:
This new code fixes the bug mentioned in the description: dragNeighbors used to reset the pointerOffset. It would make the piece you're holding suddenly snap to the grid. This would be a jump of at most 5px. This PR bumps that to 9px, so the jump gets more noticeable.

With this, the pointerOffset is retained, so instead the neighbors you're multi-selecting snap to the piece you were dragging.

Reply:
I think this makes sense, but I'm having trouble discerning the visual difference 🙈 😅 If it isn't too much work, can you send a video to illustrate the difference? If you don't have time, please say so! The game play looks good to me, so I'm ok to merge this without seeing a video.

Reply:
Oh yes I do see the difference now